server.R

```diff
@@ -1605,7 +1301,7 @@ dblclickPopupDrug <- function(
         inputId = "dblclickUnits",
         label = "Units",
         choices = c(units),
-        selected = drugDefaults$Bolus.Units[i],
+        #selected = drugDefaults()$Bolus.Units[i],
```

Comment:
remove this entirely?

Reply:
Yes it can be removed. I'm happy to see you're going through my code!

There's actually other places in the code that also make references to `i` that's not defined in local scope. This code used to work because `i` was defined as a leftover from a random for loop earlier but that for loop no longer exists. The only reason I noticed this one is because I actually noticed an error, but there must be other places.